web/src/main/java/org/mskcc/cbio/oncokb/api/websocket/CurationValidationApiController.java

```diff
@@ -341,6 +363,42 @@ public JSONArray getTruncatingMutationsNotUnderTSG() throws ApiException {
         return data;
     }
 
+    public JSONArray getActionableVariantsNotOncogenic() throws ApiException {
+        JSONArray data = new JSONArray();
+        List<Alteration> alterations = AlterationUtils.getAllAlterations();
+        List<String> allowedOncogenicities = new ArrayList<>(Arrays.asList(Oncogenicity.YES.getOncogenic(), Oncogenicity.LIKELY.getOncogenic(), Oncogenicity.RESISTANCE.getOncogenic()));
+        for (Alteration alteration : alterations) {
+            Query query = new Query(
+                null,
+                DEFAULT_REFERENCE_GENOME,
+                alteration.getGene().getEntrezGeneId(),
+                alteration.getGene().getHugoSymbol(),
+                alteration.getAlteration(),
+                null,
+                null,
+                null,
+                null,
+                alteration.getProteinStart(),
+                alteration.getProteinEnd(),
+                null);
+            IndicatorQueryResp response = IndicatorUtils.processQuery(query, null, true,null, false);
+            if (!allowedOncogenicities.contains(response.getOncogenic())) {
+                if (response.getHighestSensitiveLevel() != null || response.getHighestResistanceLevel() != null) {
+                    String hugoSymbol = alteration.getGene().getHugoSymbol();
+                    StringBuilder messageBuilder = new StringBuilder();
+                    messageBuilder.append(hugoSymbol);
+                    messageBuilder.append(" ");
+                    messageBuilder.append(alteration.getAlteration());
+                    messageBuilder.append(" is ");
+                    messageBuilder.append(response.getOncogenic());
+                    messageBuilder.append(", but is actionable.");
```

Comment:
Can we provide some additional info in terms of what is the actionable here? Maybe include the drug and level?

Reply:
My thought was that there could be multiple CTs and Treatments under a mutation, which could result in a long text. I imagine the mutation + alteration would be enough to point the curators to the right place, so they can delete all treatments.